### ISSUE TITLE
Broken Javadoc URL

### ISSUE THREAD

**Issue by NotMyFault** [apache/logging-log4j2#1284]
Hey,

It appears that a recent changed forced a change of the Javadoc URL from https://logging.apache.org/log4j/2.x/log4j-api/apidocs/ to https://logging.apache.org/log4j/2.x/javadoc/log4j-api/index.html.

Is there a particular reason why this was changed without a redirect pointing to the new location? This change affects [hundreds](https://cs.github.com/?scopeName=All+repos&scope=&q=https%3A%2F%2Flogging.apache.org%2Flog4j%2F2.x%2Flog4j-api%2Fapidocs%2F+NOT+lang%3Ahtml) of files in open source projects, now left behind broken.

Some builds may also fail because a build tool can no longer obtain the file from the old location:
```
Error fetching URL: https://logging.apache.org/log4j/2.x/log4j-api/apidocs/ (java.io.FileNotFoundException: https://logging.apache.org/log4j/2.x/log4j-api/apidocs/package-list)
```

**Comment by vy** [comment]
Thanks for the heads up @NotMyFault. I have fixed it, for now. I will bring this up in the `dev@` mailing list to decide on a permanent fix.

**Comment by vy** [comment]
Fixed in b1b687be624ca02149edd5c27c030db470dca7b0.

**Comment by NotMyFault** [comment]
> Fixed in [b1b687b](https://github.com/apache/logging-log4j2/commit/b1b687be624ca02149edd5c27c030db470dca7b0).

Thanks!
